### ISSUE TITLE
Graphical Installer rendering glitches

### ISSUE THREAD

**Issue by polarathene** [Antergos/Cnchi#504]
Latest iso, 1st time was minimal, 2nd time I used the larger gnome iso. I am installing this on a laptop with an intel iGPU and nvidia 860M card, this appears to have issues with nouveau support, I booted into the installer with the additional parameter `nouveau.modeset=0`, these errors may be due to the driver.

When using the graphical installer, I've encountered multiple rendering glitches. The first one was at the selection of how to install the OS/partitions and was effectively the two icons without any radio buttons or text to make sense of what was going on. This was problematic on the next screen with similar options either rendered broken or missing content. A 2nd attempt was better but showed some issues when using advanced option to select my partitions, one of the buttons text/label was missing when I navigated back to that screen after reviewing the following one.

**Comment by karasu** [comment]
I think this is driver related and not Cnchi's fault.

Anyway, can you post here a screenshot so I can see exactly what's happening?

**Comment by polarathene** [comment]
@karasu I've tried with `nomodeset` and `nouveau.modeset=0` but neither time could I replicate the issue. I can't quite remember if I was doing anything else, I recall Chapeau's live media having issues from using the browser over a duration of time, their tmpfs space was small and once it filled up their Gnome live environment broke horribly. I used the same usb with Antergos media still installed on it, presumably it didn't update anything once it had an internet connection? And my current install and tampering with Antergos since shouldn't affect it either correct? I guess I just got unlucky twice in a row.

I've noticed switching between Nouveau and Nvidia drivers a bit with xorg.conf causing some boot issues, when I'd manage to boot in certain changes occurred that you wouldn't think would be affected such as Gnome extensions all being disabled, date resetting to Jan 1 2009 sometimes, missing lock/suspend buttons on the top right drop down(this persists after reverting back to Nouveau). I'd still find it hard to believe these would affect the live media though.

Closing as I cannot reproduce.
